subsys/shell/Kconfig

```diff
@@ -61,12 +69,14 @@ config SHELL_DEFAULT_TERMINAL_HEIGHT
 config SHELL_ARGC_MAX
 	int "Maximum arguments in shell command"
 	range 3 255
+	default 8 if SHELL_LIGHTWEIGHT
 	default 12
 	help
 	  Maximum number of arguments that can build a command.
 
 config SHELL_TAB
 	bool "Enable the Tab button support in shell"
+	default n if SHELL_LIGHTWEIGHT
 	default y
```

Comment:
default y if !SHELL_LIGHTWEIGHT

similarly below

It seems odd to have 'default n' when that is the default anyway, until otherwise chosen

Reply:
Done